src/main/scala/temple/generate/server/go/service/dao/GoServiceDAOFunctionsGenerator.scala

```diff
@@ -42,6 +43,185 @@ object GoServiceDAOFunctionsGenerator {
       },
     )
 
+  private def generateQueryArgs(
+    operation: CRUD,
+    idAttribute: IDAttribute,
+    createdByAttribute: CreatedByAttribute,
+    attributes: ListMap[String, Attribute],
+  ): Seq[String] = {
+    val prefix  = "input"
+    lazy val id = Seq(s"$prefix.${idAttribute.name.toUpperCase()}")
+    lazy val createdBy = createdByAttribute match {
+      case EnumerateByCreator(inputName, _, _) => Seq(s"$prefix.${inputName.capitalize}")
+      case _                                   => Seq.empty
+    }
+    lazy val filteredAttributes = (attributes.collect {
+      case (name, attribute) if !attribute.accessAnnotation.contains(Annotation.ServerSet) =>
+        s"$prefix.${name.capitalize}"
+    }).toSeq
+
+    operation match {
+      case List          => createdBy
+      case Create        => id ++ createdBy ++ filteredAttributes
+      case Read | Delete => id
+      case Update        => filteredAttributes ++ id
+    }
+  }
+
+  private def generateQueryBlockErrorHandling(
+    serviceName: String,
+    operation: CRUD,
+    idAttribute: IDAttribute,
+  ): Option[String] =
+    operation match {
+      case List => Some(generateCheckAndReturnError("nil"))
```

Comment:
Personal preference is that if one arrow goes onto many lines, they all should - up to you

Reply:
Wish the formatter was a bit more opinionated

Reply:
Or I could stop being lazy and make the formatter do this for us.. I'll do that 😄